app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "NotVaccinatedError"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "OutdatedVaccineError"
+
+
+class NotWearingMaskError(Exception):
+    def __str__(self) -> str:
+        return "NotWearingMaskError"
```

Comment:
The `__str__` method for `NotWearingMaskError` should return a more descriptive message. Consider using 'Visitor is not wearing a mask.' to provide a clear error message.